### PR TITLE
Support the build with linux ddk 1.11

### PR DESCRIPTION
**Pull request by iusyk**
The new recipes to suppport the build with lddk 1.11 are created.
One feature is added: support thelist of the components to be excluded.
The varieble EXCLUDED_APIS is introduced into gles-module/gles-user-module_1.11.bb.
To exclude the builf of opencl and vulkan the following code was added into gles-user-module_1.11.bbappend (domd and domu)
EXCLUDED_APIS = "opencl vulkan"

The preferred version of GLES is changed at 1.11

Signed-off-by: Usyk Ihor <ihor.usyk@google.com>

### CODE REVIEW

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/gles-user-module_1.11.bbappend [hunk 1]
```diff
@@ -0,0 +1,6 @@
+require inc/xt_shared_env.inc
+
+BRANCH = "1.11/5516664"
+SRCREV = "${AUTOREV}"
+PREFERRED_VERSION_llvmpvr = "7.0.1"
```
Comment (otyshchenko1) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/gles-user-module_1.11.bbappend]: Do we really need this here?

File: recipes-domu/domu-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/gles-user-module_1.11.bbappend [hunk 2]
```diff
@@ -0,0 +1,6 @@
+require gles-user-module.inc
+
+BRANCH = "1.11/5516664"
+SRCREV = "${AUTOREV}"
+PREFERRED_VERSION_llvmpvr = "7.0.1"
```
Comment (otyshchenko1) [inline recipes-domu/domu-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/gles-user-module_1.11.bbappend]: same question here

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/gles-user-module_1.11.bbappend (hunk 1 above)
Comment (iusyk) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/gles-user-module_1.11.bbappend]: support of llvm is removed

File: recipes-domu/domu-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/gles-user-module_1.11.bbappend (hunk 2 above)
Comment (iusyk) [reply recipes-domu/domu-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/gles-user-module_1.11.bbappend]: support of llvm is removed

**Comment by otyshchenko1** [comment]
lgtm

**Pull request merged**